test/com/google/enterprise/adaptor/fs/WindowsFileDelegateTest.java

```diff
@@ -936,6 +937,113 @@ public boolean pushRecord(DocIdPusher.Record record) {
     delegate.destroy();
   }
 
+  @Test
+  public void testMonitorBackOffOnError() throws Exception {
+    Path file1 = newTempFile(tempRoot, "test1.txt");
+    Path file2 = newTempFile(tempRoot, "test2.txt");
+    Path file3 = newTempFile(tempRoot, "test3.txt");
+    byte[] contents = "Hello World".getBytes("UTF-8");
+
+    // Delegate with a Kernel32Ex that fails ReadDirectoryChangesW.
+    Kernel32Ex kernel32 =
+        new ChangeFailingKernel32(Kernel32Ex.INSTANCE, 0, 64, 64, 53);
+
```

Comment:
Maybe delete this blank line, and maybe also the next one, given that the comment above seems aimed at the delegate below?